src/app/give-help/law-form/law-form.component.ts

```diff
@@ -0,0 +1,142 @@
+import { ChangeDetectionStrategy, Component, ElementRef, OnInit, ViewChild } from '@angular/core';
+import { MatDialog, MatDialogRef } from '@angular/material/dialog';
+import { ActivatedRoute, Router } from '@angular/router';
+import { LawOffer, LawOfferDefinitionDTO, LawResourceService } from '@app/core/api';
+import { ConfirmCancelDialogComponent } from '@app/shared/components';
+import {
+  DIALOG_CANCEL_OFFER_CONFIG,
+  LAW_LANGUAGES,
+  LENGTH_OF_STAY,
+  MATCH_NON_DIGITS,
+  MATCH_SPACES,
+  PREFIXES,
+} from '@app/shared/consts';
+import { ALERT_TYPES, CANCEL_DIALOG_HEADERS, CorePath, Option } from '@app/shared/models';
+import { SnackbarService } from '@app/shared/services';
+import { defaults } from '@app/shared/utils';
+import { take } from 'rxjs';
+
+const HELP_KIND = Object.entries(LawOffer.HelpKindEnum).map(([key, value]) => ({
+  code: value,
```

Comment:
Czy przypadkiem code oraz label to nie ta sama wartość? Może nie ma sensu ich rozdzielać?

Reply:
Masz rację, jest to nadmiarowe, dzięki.